draft-ietf-scitt-architecture.md

```diff
@@ -939,7 +940,7 @@ In particular, they cannot "frame" an Issuer or a Transparency Service for Signe
 
 #### Append-only Log
 
-If a Transparency Service is honest, then a Transparent Statement including a correct Receipt ensures that the Transparent Statement passed its Registration Policy and was recorded appropriately.
+If a Transparency Service is honest, then a Transparent Statement including a correct Receipt ensures that the associated Signed Statement passed its Registration Policy and was recorded appropriately.
```

Comment:
```suggestion
A Transparent Statement including a Receipt ensures that the associated Signed Statement passed its Registration Policy and was recorded appropriately.
```

Reply:
This section talks about Append Only log, so the `text` about honesty makes sense here?

Reply:
thats fine, I don't think the text is wrong, I just don't think it's needed here.